### DISCUSSION TITLE
RfcLogWrite: could not open //./dev_rfc_4161208.log. Trace directory set to ./

### DISCUSSION THREAD

**Discussion by sebfeldm** [Rocket-Search/sap-mon#3] [General]
We are trying to implement sap_mon in checkmk 2.0. I am running the plugin under "Integrate Nagios plugins" and while the shell output is fine, the output of the checkmk core for:
`sap_mon -check -username=RFC_TEST -password=Test123456 -hostname=saplnx -sid=AL1 -sysnum=01 -client=100 -monitor=='AL1\saplnx_AL1_01\OperatingSystem\CPU\CPU_Utilization'`
is:
`RfcLogWrite: could not open //./dev_rfc_4159813.log. Trace directory set to ./2`
should be:
`2`

While the output should only be for example "2". As I am not able to fix the pathing for the checkmk core to make it able to write the log, is there any parameter to disable RfcLogWrite?

**Comment by Rocket-Search** [comment]
Hello seebus,

thanks for your reply.
Unfortunately there is no parameter to disable to write logs at runtime.
The only way is to change the code.
Please feel free to adjust the code.
For a little donation I can change the code as you wish.
If yes, please send a email to software.moore@gmail.com .

Best regards
T. Reineck
